android-components/components/ui/autocomplete/src/main/java/mozilla/components/ui/autocomplete/InlineAutocompleteEditText.kt

```diff
@@ -621,9 +628,6 @@ open class InlineAutocompleteEditText @JvmOverloads constructor(
 
             override fun setComposingText(text: CharSequence, newCursorPosition: Int): Boolean {
                 return if (removeAutocompleteOnComposing(text)) {
-                    if (isSonyKeyboard) {
```

Comment:
Do we no longer the device specific check here? I don't have a sony to test this.

Reply:
`restartInput()` call is moved inside `removeAutocompleteOnComposing(text: CharSequence)` and called every time regardless of current input method.

https://github.com/mozilla-mobile/firefox-android/blob/44f9d15fdad28695548138ec7de0feb952752631/android-components/components/ui/autocomplete/src/main/java/mozilla/components/ui/autocomplete/InlineAutocompleteEditText.kt#L607-L617

Because there is no API to notify IME that composing text was changed, apart from `restartInput()` entirely.